posts/2021-07-22-cloud-hosted-guides-post.adoc

```diff
@@ -65,23 +59,26 @@ image::/img/blog/OL_Cloud-hosted-3.png[Image 3,width=70%,align="center"]
 
 Once you are logged in you will then be directed to the landing page for the guide you wish to learn. Simply click on the “Access cloud-hosted guide” button and after a few minutes your environment should be set-up and ready to use.
 
-== The Basics of the Environment
+== What is the Skills Network and why are we using it with our guides?
 
-The environment is quite easy to use with your instructions on the left and your IDE with a terminal on the right. Each guide has different steps with instructions that you can copy straight to your clipboard by clicking on the image:/img/blog/OL_Cloud-hosted-4.png[] button. You can change the font, font size and change the instructions from light to dark mode using the panel above the instructions frame. You can also resize the instruction and IDE frames to suit your screen by dragging on the edge of the frames. 
+The Skills Network is a learning environment that runs through a user’s browser. It runs in containers that are deployed to OpenShift in the IBM Cloud. This allows us not only to teach users how to build great cloud-native applications but also demonstrates that in practice with how the environment is deployed.
```

Comment:
Can we re-work this?  We don't need to mention SNL this way.

Reply:
Done